fast_llm/utils.py

```diff
@@ -87,6 +87,10 @@ class Assert:
     def eq(x, *args):
         for arg in args:
             assert x == arg, f"{x} != {arg}"
+    
+    @staticmethod
+    def eq_with_msg(x, y, msg=""):
```

Comment:
Don' think it's worth a separate method. Let's either add the optional argument to Assert.eq, or just white the error inline.